scripts/unit-tests.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+uv python install 3.10.16
```

Comment:
It may be worth doing something smarter to prevent multiple installations of Python 3.10 on a user system (for example, I tried this when I already had a Python 3.10.16 system installation and it added another) - something like the following
```suggestion
uv python find 3.10.16
IS_PYTHON=$?
if [ $IS_PYTHON -eq 0 ]; then
     uv python install 3.10.16
fi
```